unified-runtime/source/adapters/level_zero/common.hpp

```diff
@@ -610,3 +610,37 @@ extern thread_local int32_t ErrorAdapterNativeCode;
                                       int32_t AdapterErrorCode);
 
 #define L0_DRIVER_INORDER_MIN_VERSION 29534
+// Definitions for the External Semaphore Extension
+
+#ifndef ZE_INTEL_EXTERNAL_SEMAPHORE_EXP_NAME
```

Comment:
You should not need any of the external semaphore definitions. This looks to be a bad rebase.